### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by JBX2010** [AngelDTF/OpenParsec#1]
I have a IPad Pro 12,9 inch. Is it possible to install this client?

**Comment by JBX2010** [comment]
Okay, thanks ... Soo there will be no build releases anytime soon?

**Comment by AngelDTF** [comment]
Correct. I'll be looking into making releases whenever I get further into development, and I cannot guarantee when that'll be.

**Comment by SovietGundam** [comment]
😶1. Thank you!

2. Uuuh future clearer instructions would be awesome, and is there anyway to do this without a Mac or MacOS. (Only have Ipad)

**Comment by AngelDTF** [comment]
At the moment, there unfortunately is no other way to build the application. As I am not enrolled in the Apple Developer Program, I am unsure of how I could distribute a built application. I’ll need to do more research to see if there’s any workarounds for making IPA files, and even then you’ll need a way to side-load the application.

**Comment by bslatyer** [comment]
I can build an unsigned IPA to distribute.

Just let me know 😄 

Thanks,

Braeden

**Comment by marco21187** [comment]
can you make unsigned IPA for me :D

**Comment by Aarav5255** [comment]
Please make the ipa

**Comment by bslatyer** [comment]
I'll make it today - just been really busy.

**Comment by bslatyer** [comment]
Here you go y'all - remember to resign it with your own Apple Developer certificates 
[OpenParsec 2022-12-22 09-01-59.zip](https://github.com/AngelDTF/OpenParsec/files/10281808/OpenParsec.2022-12-22.09-01-59.zip)

**Comment by Aarav5255** [comment]
Hey thanks for the ipa. Im having problems signing in. I keep getting kicked out.

**Comment by marco21187** [comment]
seems like we were having same issue
[OpenParsec-2022-12-22-070441.zip](https://github.com/AngelDTF/OpenParsec/files/10282358/OpenParsec-2022-12-22-070441.zip)

**Comment by AngelDTF** [comment]
Sorry for being gone, was really sick for a couple weeks. I'll try to take a look at the issue when I'm free (doing stuff with family for the holidays). In the meantime, if there's any more details you can give in regards to the issue, please inform me and I'll see what I can do.

**Comment by jarguard** [comment]
I hope you get better 😁

**Comment by Aarav5255** [comment]
Any solutions?

**Comment by AngelDTF** [comment]
My apologies, but I've been unable to diagnose any issues as I thought my condition would've improved enough by now for me to get back to programming, however I'm still not fully recovered and cannot spend much time on my PC. I have a chronic medical condition that comes and goes, but I will be seeing a doctor soon, and hopefully within a couple more weeks I'll have improved enough to be able to work on this again.

**Comment by bslatyer** [comment]
I will have a look when I get some free time to see if I can figure out the current issues

**Comment by Aarav5255** [comment]
> My apologies, but I've been unable to diagnose any issues as I thought my condition would've improved enough by now for me to get back to programming, however I'm still not fully recovered and cannot spend much time on my PC. I have a chronic medical condition that comes and goes, but I will be seeing a doctor soon, and hopefully within a couple more weeks I'll have improved enough to be able to work on this again.

No worries, please take your time. Hope you get well soon

**Comment by Aarav5255** [comment]
Still no fixes?

**Comment by SovietGundam** [comment]
I got the .ipa Signed and installed onto my Ipad Pro M1 11-inch.

It launches, and does sign into my parsec account and it does look for errors in my sign-in, however I cannot connect to any device using the app. I get an [Error connecting to host](https://support.parsec.app/hc/en-us/articles/115002624531-Error-Codes-6-Your-Request-to-Access-the-Server-Was-Denied-#:~:text=This%20error%20happens%20when%20you,or%20by%20pressing%20CTRL%2BF1.). According to Parsec this is because the request was never accepted but I don't believe the request reaches my PC. 

I'm pretty new to IOS apps, and have NO idea how to troubleshoot this. Any advice?

**Comment by SovietGundam** [comment]
Tried using the app again today, and it suddenly worked  ¯\_(ツ)_/¯

I changed nothing since last time. it just decided this time it will work.

But I cannot use touch to interact, I sadly do not currently have access to devices I can connect to my device(keyboard/mouse/controller) when I get my controller back, I'll try using the controller.

**Comment by Aarav5255** [comment]
Can someone give me the ipa pls?

**Comment by MaximVerhaert** [comment]
When building te project on my ipad, I get a black screen with the OpenParsec logo. How can I login etc?

**Comment by qnblackcat** [comment]
@SovietGundam could you share the IPA?
